group_vars/all/apps.yml

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
I should annotate/comment this file.

Reply:
I'm not sure this file should be versionned. Maybe I'm wrong but I understand that I will have to edit it if I want to add a third party application. If I modify it, it will appear in the list of modified files in my git status and I don;t think we want this.

Reply:
You can safely override default variables for a customer/env_type with your own apps repository.